inst/rmd/h/default.html

```diff
@@ -130,6 +133,26 @@
 h6 {
   font-size: 12px;
 }
+$else$
+/*
+BS3 -> BS4 saw a removal of margin-top which is a pretty jarring change for Rmd
+https://github.com/twbs/bootstrap/blob/f17f882d/dist/css/bootstrap.css#L1219-L1250
+https://github.com/twbs/bootstrap/blob/4de4874e/scss/_reboot.scss#L82-L89 */
+h1, h2, h3 { margin-top: 20px; }
+h4, h5, h6 { margin-top: 10px; }
+
+/* Pandoc uses align attribute to align content but BS4 squashes that */
```

Comment:
Maybe this should as a part of `bootstraplib::bs_sass()`? flexdashboard suffers from the same problem